docs/UserGuide.md

```diff
@@ -204,7 +204,9 @@ Examples:
 
 ### Undo previous entry : `undo`
 
-Undo the previous command, stackable. (Able to keep undo-ing till there are no commands left to be undone)
+Undo the previous command, stackable (all commands that were successfully executed previously can be continuously undone until there are no commands left to be undone).
```

Comment:
This updated formulation for the undo seems clearer. Maybe we can follow the same format for the redo part of the UG?